metadata.json

```diff
@@ -74,6 +74,13 @@
       "operatingsystemrelease": [
         "18"
       ]
+    },
+    {
+      "operatingsystem": "FreeBSD,
```

Comment:
please add the missing `"` and fix the alignment in the lines below (I highly recommend using soft tabs)

Reply:
Hi, my bad. The editor decided to insert some quotes by itself which I overlooked. Should be ok now. Hard tabs are gone too